pkg/operator/controller/ingress/controller_test.go

```diff
@@ -317,3 +317,60 @@ func TestTLSProfileSpecForIngressController(t *testing.T) {
 		}
 	}
 }
+
+func TestValidateHTTPHeaderBufferValues(t *testing.T) {
+	testCases := []struct {
+		description      string
+		httpHeaderBuffer operatorv1.IngressControllerHTTPHeaderBuffer
+		valid            bool
+	}{
+		{
+			description:      "empty httpHeaderBuffer",
+			httpHeaderBuffer: operatorv1.IngressControllerHTTPHeaderBuffer{},
+			valid:            true,
+		},
+		{
+			description: "valid, non-empty httpHeaderBuffer",
+			httpHeaderBuffer: operatorv1.IngressControllerHTTPHeaderBuffer{
+				HeaderBufferBytes:           32768,
+				HeaderBufferMaxRewriteBytes: 8192,
+			},
+			valid: true,
+		},
+		{
+			description: "invalid, non-empty httpHeaderBuffer",
```

Comment:
The wording confused me; we talked offline so I think you're going to propose "field is set??"

Reply:
Changed wording. Should be good now. Went with `values` phrasing instead of `field is set`.